README.md

```diff
@@ -2,15 +2,15 @@
 
 <div align="center">
 
-  <h1><b>TDD</b></h1>
+  <h1><b>TDD-RUBY-SPEC PROJECT</b></h1>
 
 </div>
 
 <!-- TABLE OF CONTENTS -->
 
 ## 📗 Table of Contents
 
-- [📖 TDD ](#-about-project-)
+- [📖 TDD-RUBY-SPEC PROJECT ](#-about-project-)
```

Comment:
- [ ] As mentioned earlier, kindly ensure that this link address is the same as the name given to the link element in line 31 below. The link does not redirect as of the moment. 🙏 

## Sample

<img width="260" alt="Untitled" src="https://github.com/AbuTalha3/tdd-ruby-spec/assets/103386183/a244f42d-fec3-411b-aaa2-a13b13dfa912">